tests/setup/mod.rs

```diff
@@ -99,17 +99,17 @@ pub mod nigiri {
     }
 
     pub fn start() {
-        START_LOGGER_ONCE.call_once(|| {
+        INIT_SETUP_ONCE.call_once(|| {
             SimpleLogger::init(simplelog::LevelFilter::Debug, simplelog::Config::default())
                 .unwrap();
-        });
 
-        // TODO: Optimization, do not restart nigiri if
-        // `jq -r .ci  ~/.nigiri/nigiri.config.json` is true.
-        if env::var("RUNNING_ON_CI").is_err() {
-            debug!("NIGIRI stopping ...");
-            stop();
-        }
+            // TODO: Optimization, do not restart nigiri if it is already running in --ci mode:
+            //       `jq -r .ci  ~/.nigiri/nigiri.config.json` == true.
+            if env::var("RUNNING_ON_CI").is_err() {
+                debug!("NIGIRI stopping ...");
+                stop();
+            }
+        });
```

Comment:
This makes tests run faster, but it means that nigiri's state is kept between different integration tests. I think it's best if each integration test is independent of the others, so I would keep stopping nigiri before every INT test.

Reply:
Please note that this has been done locally only anyways: `if env::var("RUNNING_ON_CI").is_err()`. I perceived this more as a measure to ensure Nigiri is not running in a non-ci mode at the point in time when test are being started (which isn't the case when tests are run on CI anyways).

It is correct that Nigiri's state is being kept during all tests for now (and even before all of my changes, but only on CI, while my change produces the same behaviour locally as on CI), but the Lipa-node keeps forgetting it's old channels because its storage gets resetted between every test. That's why the tests all end up being successful.

I'll go ahead and add the additional functionality of resetting Nigiri after every test run.

Reply:
Changed here: 5ea7f513005ffef4fb557f364aeae2d80e5f0fda